build.gradle.kts

```diff
@@ -54,7 +55,9 @@ tasks {
         options.release.set(8)
     }
     withType<KotlinCompile>().configureEach {
-        kotlinOptions.jvmTarget = "1.8"
+        compilerOptions {
+            jvmTarget = JvmTarget.JVM_1_8
+        }
```

Comment:
This is a minor change that resolves compilation warnings. No effect on the compilation result.